_layouts/sign_up_form.html

```diff
@@ -0,0 +1,35 @@
+{% include head.html %}
+
+<div class="container">
+    {% include header.html %}
+
+<h1>Hack Brexit Mailing List</h1>
+
+<p>Sign up on our Hack Brexit contact form for us to keep you updated with all of the latest news </p>
+
+<form method="POST" action="http://formspree.io/hackbrexit@gmail.com">
+
+<form class="form-inline">
```

Comment:
Nested form tags is not correct. This is the 2nd one, you can combine them both.